java/src/org/openqa/selenium/json/JsonOutput.java

```diff
@@ -350,35 +383,44 @@ public JsonOutput endArray() {
   }
 
   /**
-   * @param value
-   * @return
+   * Serialize the specified Java object as a JSON value.<br>
+   * <b>NOTE</b>: This method limits traversal of nested objects to the default
+   * {@link #MAX_DEPTH maximum depth}.
+   *
+   * @param value Java object to serialize
+   * @return this {@link JsonOutput} object
    */
   public JsonOutput write(Object value) {
     return write(value, MAX_DEPTH);
   }
 
   /**
-   * @param value
-   * @param maxDepth
-   * @return
-   */
+   * Serialize the specified Java object as a JSON value.
+   *
+   * @param value Java object to serialize
+   * @param maxDepth maximum depth of nested object traversal
+   * @return this {@link JsonOutput} object
+  */
   public JsonOutput write(Object value, int maxDepth) {
     return write0(value, maxDepth, maxDepth);
   }
 
   /**
-   * @param input
-   * @param maxDepth
-   * @param depthRemaining
-   * @return
+   * Serialize the specified Java object as a JSON value.
```

Comment:
We can omit comments for private methods. The question to ask is "does this comment add extra value to people attempting to maintain the code?" If you think the answer is "yes", then please do leave this javadoc in.

Reply:
This particular private method probably doesn't benefit much from the JavaDoc header.